include/dlaf/blas/tile.h

```diff
@@ -137,22 +137,31 @@ DLAF_MAKE_GPUBLAS_SYHE_OP(Her2k, r2k);
 
 DLAF_MAKE_GPUBLAS_SYHE_OP(Herk, rk);
 
-#if defined(DLAF_WITH_HIP)
+#if defined(DLAF_WITH_CUDA)
+DLAF_MAKE_GPUBLAS_OP(Trmm, trmm);
+#elif defined(DLAF_WITH_HIP)
+
 #if defined(__clang__)
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
 #elif defined(__GNUC__)
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #endif
-#endif
+
+// TODO: What will be the upper bound?
+#if HIP_VERSION < 50000000 || 50700000 <= HIP_VERSION
```

Comment:
I would probably make this simply
```suggestion
#if HIP_VERSION < 50000000
```
for now...